mantidproject/mantid
# Enable focus sum of files mode for multi run focusing

fixes #14835

**This should be merged after the following pull request has been merged #14830**
- [ ] _#14830 merged_

Follow up from #14579 branch, the `Focus Sum Of Files` option has been enabled and now there are two different options of focusing mode under `Engineering Diffraction/Focus tab`. 

**To Test**
- Code Check
- When multi-run given with `Focus Sum Of Files`, the Focusing process should only be carried out once or twice; depending on the number of banks selected
- Appropriate message provided when single file given instead of multiple for `Focus Sum Of Files`
- Make sure the correct run-number is utilised while focusing instead just reading from basic focus run-number
